client/src/context/useBookingContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { useContext, createContext, FunctionComponent, useState, useEffect } from 'react';
+import { Booking } from '../interface/Booking';
+import sortedBookings from '../helpers/utils/sortBookings';
+
+interface IBookingContext {
```

Comment:
```suggestion
interface BookingContext {
```
We don't need to prefix interfaces with 'I' in JS :)